testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/OID4VCSdJwtIssuingEndpointTest.java

```diff
@@ -391,7 +398,8 @@ protected void handleCredentialResponse(CredentialResponse credentialResponse) t
             assertTrue("The credentials should include the email claim.", disclosureMap.containsKey("email"));
             assertEquals("email claim incorrectly mapped.", disclosureMap.get("email").get(2).asText(), "john@email.cz");
 
-        }
+            assertNotNull("Test credential shall include an iat claim.", jsonWebToken.getIat());
+            assertNotNull("Test credential shall include an nbf claim.", jsonWebToken.getNbf());        }
```

Comment:
I think that It might be better to check the value of `jsonWebToken.getNbf()` in a reasonable range. WDYT?

Reply:
Same as IAT. As value depend on privacy policy of issuer.